package.json

```diff
@@ -3,69 +3,70 @@
   "version": "0.0.0",
   "dependencies": {
     "@hashicorp/mktg-global-styles": "^4.0.0",
-    "@hashicorp/mktg-logos": "^1.0.2",
-    "@hashicorp/nextjs-scripts": "^19.0.3",
-    "@hashicorp/sentinel-embedded": "^0.0.9",
-    "classnames": "^2.2.6",
+    "@hashicorp/mktg-logos": "^1.2.0",
+    "@hashicorp/sentinel-embedded": "^0.0.14",
+    "classnames": "^2.3.1",
     "formik": "^2.2.9",
-    "next": "^10.0.6",
-    "next-remote-watch": "^0.3.0",
+    "next": "^11.1.0",
+    "next-remote-watch": "^1.0.0",
     "node-cache": "^5.1.2",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
     "semver": "^7.3.5",
-    "swingset": "^0.10.0"
+    "swingset": "^0.10.1"
   },
   "devDependencies": {
-    "@hashicorp/platform-cli": "0.1.0",
+    "@hashicorp/platform-cli": "^1.1.0",
     "@hashicorp/platform-cms": "^0.1.0",
-    "@hashicorp/platform-code-highlighting": "0.1.0",
-    "@hashicorp/platform-docs-mdx": "0.1.0",
-    "@hashicorp/platform-nextjs-plugin": "0.1.0",
-    "@hashicorp/platform-types": "0.1.0",
+    "@hashicorp/platform-code-highlighting": "0.1.1",
+    "@hashicorp/platform-docs-mdx": "0.1.1",
+    "@hashicorp/platform-nextjs-plugin": "0.2.0",
+    "@hashicorp/platform-types": "^0.1.0",
     "@hashicorp/platform-util": "0.1.0",
-    "@octokit/core": "^3.3.1",
-    "@testing-library/jest-dom": "^5.11.6",
-    "@testing-library/react": "^11.2.2",
-    "@testing-library/user-event": "^13.1.1",
-    "@types/react": "^17.0.3",
-    "dotenv": "^8.2.0",
-    "eslint": "^7.18.0",
-    "husky": "^4.3.8",
+    "@octokit/core": "^3.5.1",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/react": "^17.0.16",
+    "dotenv": "^10.0.0",
+    "eslint": "^7.32.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
+    "jest": "^27.0.6",
     "jest-raw-loader": "^1.0.1",
-    "lerna": "^3.22.1",
-    "lint-staged": "^10.5.3",
-    "prettier": "^2.2.1",
-    "typescript": "^4.3.2"
+    "lerna": "^4.0.0",
+    "prettier": "^2.3.2",
+    "simple-git-hooks": "^2.5.1",
+    "typescript": "^4.3.5"
+  },
+  "engines": {
+    "npm": ">=7.0.0",
+    "node": ">=10.0.0"
+  },
+  "resolutions": {
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap -- --no-optional",
-    "bs": "npm run bootstrap",
```

Comment:
Removing references to `lerna bootstrap` as everything is handled via `npm` now.